lib/nano_rpc/client.rb

```diff
@@ -59,8 +60,16 @@ def rpc_post(params)
       data
     end
 
+    def headers
+      if @auth.nil?
```

Comment:
Would suggest rewriting this as 

```ruby
headers = { 'Content-Type' => 'json' }
headers.merge('Authorization' => auth) unless auth.nil?
headers
```

(1) Single quotes to make Ruboco happy
(2) Removes indenting
(3) Removes repetition
(4) Uses accessor instead of instance var lookup

Reply:
Good idea. (I think you want `merge!` though.)